packages/rx_bloc_cli/test/extensions/arg_results_extensions_test.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:args/args.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:rx_bloc_cli/src/extensions/arg_results_extensions.dart';
+import 'package:rx_bloc_cli/src/models/command_arguments.dart';
+import 'package:rx_bloc_cli/src/models/errors/command_usage_exception.dart';
+import 'package:rx_bloc_cli/src/models/realtime_communication_type.dart';
+import 'package:test/test.dart';
+import 'arg_results_extensions_test.mocks.dart';
+
+@GenerateNiceMocks([
+  MockSpec<ArgResults>(),
+])
+void main() {
+  late MockArgResults sut;
```

Comment:
Is there a better word than `sut` for this object.

Reply:
It's an abbreviation for "System under test", which is commonly used as a name in unit tests

Reply:
Thanks